README.md

```diff
@@ -1,100 +1,240 @@
-# Hubble
+Table of Contents
+=================
 
-An alternate version of Hubblestack which can be run without an existing
-SaltStack infrastructure.
+   * [HUBBLE](#hubble)
+      * [Packaging / Installing](#packaging--installing)
+         * [Installing using setup.py](#installing-using-setuppy)
+         * [Building standalone packages (CentOS)](#building-standalone-packages-centos)
+         * [Building standalone packages (Debian)](#building-standalone-packages-debian)
+         * [Buidling Hubble packages through Dockerfile](#buidling-hubble-packages-through-dockerfile)
+      * [Nova](#nova)
+         * [Usage](#usage)
+         * [Configuration](#configuration)
+      * [Nebula](#nebula)
+         * [Usage](#usage-1)
+         * [Configuration](#configuration-1)
+      * [Pulsar](#pulsar)
+         * [Usage](#usage-2)
+         * [Configuration](#configuration-2)
+            * [Excluding Paths](#excluding-paths)
+            * [Pulsar topfile top.pulsar](#pulsar-topfile-toppulsar)
 
-# Packaging / Installing
+# HUBBLE
 
-## Installing using setup.py
+Hubble is a modular, open-source, security & compliance auditing framework which is built on SaltStack. It is alternate version of Hubblestack which can be run without an existing SaltStack infrastructure. Hubble provides on-demand profile-based auditing, real-time security event notifications, alerting and reporting. It also reports the security information to Splunk. This document describes installation, configuration and general use of Hubble.
 
-```bash
+## Packaging / Installing
+### Installing using setup.py
+```sh
 sudo yum install git -y
 git clone https://github.com/hubblestack/hubble
 cd hubble
 sudo python setup.py install
 ```
+Installs a hubble "binary" into `/usr/bin/`.
 
-Installs a `hubble` "binary" into `/usr/bin/`.
+A config template has been placed in `/etc/hubble/hubble`. Modify it to your specifications and needs. You can do `hubble -h` to see the available options.
 
-## Building standalone packages (CentOS)
+The first two commands you should run to make sure things are set up correctly are `hubble --version` and `hubble test.ping`.
 
-```bash
+### Building standalone packages (CentOS)
```

Comment:
Can you please remove these two sections that use the build_rpms.sh and build_debs.sh scripts? I don't think they're working at this point so we want to point people to the dockerfiles.